docs/src/app/(public)/(content)/react/utils/use-render/demos/data-attributes/css-modules/index.module.css

```diff
@@ -0,0 +1,38 @@
+.Button {
```

Comment:
Can we make the counter text width fixed, e.g. limit it to support up to 3 digits, so that it doesn't jump when we click the button.